registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/mapper/PacketInfoMapper.java

```diff
@@ -141,7 +146,7 @@ private static String[] getLanguages(JsonValue[] jsonNode, StringBuilder languag
 	 * @return the list
 	 */
 	public static List<IndividualDemographicDedupeEntity> converDemographicDedupeDtoToEntity(
-			IndividualDemographicDedupe demoDto, String regId) throws NoSuchAlgorithmException {
+			IndividualDemographicDedupe demoDto, String regId,Boolean istrim) throws NoSuchAlgorithmException {

```

Comment:
We are changing an existing public method from a different jar depdendency project. I think we should create this as a overrided method to avoid breaking any customizations from the country side.

Reply:
Done

Reply:
can we not avoid the duplicate code and simply call withTrim method from the original method ?